org-formation/_scp.yaml

```diff
@@ -126,3 +161,19 @@
               - 'route53domains:RegisterDomain'
               - 'route53domains:TransferDomain'
             Resource: '*'
+
+  # from https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_policies_scps_examples_general.html#example-scp-deny-region
+  RestrictLeaveOrganizationSCP:
+    Type: OC::ORG::ServiceControlPolicy
+    Properties:
+      PolicyName: RestrictLeaveOrganization
+      Description: Administrators of member accounts can't remove their accounts from the organization
+      PolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          - Sid: DenyDnsRegistration
```

Comment:
Could you confirm? That does not match the description and we need to be able to register / transfer domains (or are you planning to remove the permissions in all accounts except sageit?).

Reply:
oops, did copy/paste without updating description

Reply:
fixed